drivers/docker/driver.go

```diff
@@ -456,6 +462,10 @@ CREATE:
 		// There is still a very small chance this is possible even with the
 		// coordinator so retry.
 		return nil, nstructs.NewRecoverableError(createErr, true)
+	} else if isDockerTransientError(createErr) && attempted < 5 {
+		attempted++
+		time.Sleep(1 * time.Second)
```

Comment:
shouldn't we exponentially backoff here?

Reply:
:+1: or even just `time.Sleep(attempted * time.Second)`